lib/Scheduler/sched_region.cpp

```diff
@@ -77,54 +74,49 @@ void SchedRegion::CmputAbslutUprBound_() {
   abslutSchedUprBound_ = dataDepGraph_->GetAbslutSchedUprBound();
 }
 
-FUNC_RESULT SchedRegion::FindOptimalSchedule( //TODO: CHIPPIE: Add helper functions as private functions in the class.
+// FIXME: Heuristic cost and length is being returned by reference
```

Comment:
Question: -1 is reserved for invalid value or something, right? Could initialize it to that, then the calling function can just check it before using it.

Reply:
Sounds like a good idea. Updated.